app/src/main/java/com/pajato/android/gamechat/chat/fragment/ChatShowGroupsFragment.java

```diff
@@ -178,8 +178,6 @@ private int getTitleResId() {
         Account account = AccountManager.instance.getCurrentAccount();
         if (account == null)
```

Comment:
Fwiw, I tend to prefer, since it is a one-liner:
   return account == null ? R.string.app_name : R.string.GroupsToolbarTitle;